### PR TITLE
Add randint op support

### PR DESCRIPTION
**Pull request by stu1130**
## Channel for questions ##
Before or while filing an issue please feel free to join our [Slack channel](https://join.slack.com/t/deepjavalibrary/shared_invite/zt-ar91gjkz-qbXhr1l~LFGEIEeGBibT7w) to get in touch with development team, ask questions, find out what's cooking and more!

## Description ##
(Brief description of what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- [ ] Code is well-documented: 
    - For user-facing API changes, Java doc has been updated. 
    - For new examples, README.md is added to explain the what the example does.
- [ ] To the my best knowledge, [examples](https://github.com/awslabs/djl/tree/master/examples) and [jupyter notebooks](https://github.com/awslabs/djl/tree/master/jupyter) are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, Java doc)
- [ ] Feature2, tests, (and when applicable, Java doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: api/src/main/java/ai/djl/ndarray/NDManager.java [hunk 1]
```diff
@@ -1061,6 +1061,54 @@ default NDArray linspace(float start, float stop, int num, boolean endpoint, Dev
...
+    /**
+     * Return random integers from low (inclusive) to high (exclusive).
+     *
+     * @param low Lowest (signed) integers to be drawn from the distribution
+     * @param high one above the largest (signed) integer to be drawn from the distribution
+     * @param shape the {@link Shape} of the {@link NDArray}
+     * @param dataType the {@link DataType} of the {@link NDArray}
+     * @return the drawn samples {@link NDArray}
+     */
+    default NDArray randomInt(int low, int high, Shape shape, DataType dataType) {
```
Comment (frankfliu) [inline api/src/main/java/ai/djl/ndarray/NDManager.java:1085]: If randomInt can take dataType as input, we should not provide randomLong,
I think we only need:
NDArray randomInteger(long low, long high, Shape shape, DataType dataType);

File: api/src/main/java/ai/djl/ndarray/NDManager.java [hunk 2]
```diff
@@ -1061,6 +1061,17 @@ default NDArray linspace(float start, float stop, int num, boolean endpoint, Dev
         return newSubManager(device).linspace(start, stop, num, endpoint);
     }
 
+    /**
+     * Return random longs from low (inclusive) to high (exclusive).
```
Comment (frankfliu) [inline api/src/main/java/ai/djl/ndarray/NDManager.java:1065]: Returns random integer values

File: api/src/main/java/ai/djl/ndarray/NDManager.java [hunk 3]
```diff
@@ -1061,6 +1061,17 @@ default NDArray linspace(float start, float stop, int num, boolean endpoint, Dev
         return newSubManager(device).linspace(start, stop, num, endpoint);
     }
 
+    /**
+     * Return random integer values from low (inclusive) to high (exclusive).
```
Comment (frankfliu) [inline api/src/main/java/ai/djl/ndarray/NDManager.java:1065]: Returns

**Review by frankfliu — APPROVED** [review]

**Review by frankfliu — APPROVED** [review]
